packages/cli/src/commands/init.ts

```diff
@@ -0,0 +1,49 @@
+import type { CommandModule, InferredOptionTypes } from "yargs";
+import { loadConfig } from "@latticexyz/config/node";
+import { World as WorldConfig } from "@latticexyz/world";
+import { getRpcUrl } from "@latticexyz/common/foundry";
+import { Hex, createWalletClient, http } from "viem";
+import { getWorldDeploy } from "../deploy/getWorldDeploy";
+import { writeDeploymentResult } from "../utils/writeDeploymentResult";
+
+const verifyOptions = {
+  configPath: { type: "string", desc: "Path to the MUD config file" },
```

Comment:
what's the expected behavior if you already have a config?

I thought the goal of `mud init` was to generate a mud config given a world address?

Reply:
Good point, the config will ultimately be generated as part of this!